prisma/seed.js

```diff
@@ -0,0 +1,210 @@
+        console.log("team", team)
+        teams.push(team);
+    }
+    return teams;
+
+}
+
+const createParticipants = async (season, competitors, teams) => {
+    const participants = [];
+
+    for (let i = 0; i < competitors.length; i++) {
+        const participant = await prisma.participant.create({
+            data: {
+                season: {
+                    connect: {
+                        id: season.id
+                    }
+                },
+                competitor: {
+                    connect: {
+                        id: competitors[i].id
+                    }
+                },
+                team: {
+                    connect: {
+                        id: teams[Math.floor(Math.random() * MAX_TEAMS_TO_GENERATE)].id
+                    }
+                },
+            }
+        })
+        console.log("participant", participant);
+        participants.push(participant);
+    }
+    return participants
+}
+
+const createRounds = async (season) => {
+    const rounds = [];
+    for (let i = 0; i < MAX_ROUNDS_TO_GENERATE; i++) {
+        const fakeRound = createFakeRound();
+        const round = await prisma.round.create({
+            data: {
+                title: fakeRound.title,
+                startsAt: fakeRound.startsAt,
+                season: {
+                    connect: {
+                        id: season.id
+                    }
+                }
+            }
+        })
+        console.log("round", round);
+        rounds.push(round);
+    }
+    return rounds;
+
+}
+
+const createPlacements = async (participants, rounds) => {
+    const placements = [];
+
+    for (let i = 0; i < rounds.length; i++) {
+        const positions = Array.from({ length: participants.length }, (_, i) => i + 1); //fills an array up to max of participants [1,2,3 ... participants.length]
+        for (let j = 0; j < participants.length; j++) {
+            const fakePosition = createFakePlacement(positions);
+            const placement = await prisma.placement.create({
+                data: {
+                    position: fakePosition.position,//add fake position
+
+                    participant: {
+                        connect: {
+                            id: participants[j].id
+                        }
+                    },
+                    round: {
+                        connect: {
+                            id: rounds[i].id
+                        }
+                    },
+                }
+            });
+            const index = positions.indexOf(fakePosition.position);
+            positions.splice(index, 1);
+            console.log("placement", placement);
+            placements.push(placement);
+        }
+    }
+    return placements
+}
+
+const createPositionMappings = async (season, participants) => {
+    const positions = Array.from({ length: participants.length }, (_, i) => i + 1); //fills an array up to max of participants [1,2,3 ... participants.length]
```

Comment:
I've seen this same line twice now, good candidate to be moved into its own function!